utilities/trajectory_generator.py

```diff
@@ -0,0 +1,48 @@
+from wpimath.geometry import Pose2d
+from typing import List
+
+from utilities.scalers import scale_value
+
+
+def totalLength(waypoints: List[Pose2d]) -> float:
```

Comment:
nit: use snake case instead of camel case - the rest of our own functions are snake case